app/src/main/res/layout/fragment_edit_user_profile.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
do you need this fragment?
I saw you used the registration fragment

Reply:
Yes, it is needed as I recall.
It's the same as edit a restaurant page - it is needed only for the navigation graph.
The design is extending the viewing page (the same as he taught in the class)